indy-besu/vdr/src/client/implementation/web3/signer.rs

```diff
@@ -17,7 +18,11 @@ pub struct Web3Signer<'a> {
 
 impl<'a> Web3Signer<'a> {
     pub fn new(account: Address, signer: &'a Box<dyn Signer>) -> Web3Signer<'a> {
-        Web3Signer { account, signer }
+        let signer = Web3Signer { account, signer };
+
+        debug!("Created new signer");
```

Comment:
Can we log additional information about the created signer?

Reply:
Added info about address